phpunit.xml.dist

```diff
@@ -5,6 +5,9 @@
          beStrictAboutOutputDuringTests="true"
          displayDetailsOnPhpunitDeprecations="true"
          displayDetailsOnTestsThatTriggerDeprecations="true"
+         displayDetailsOnTestsThatTriggerNotices="true"
+         displayDetailsOnTestsThatTriggerWarnings="true"
+         displayDetailsOnTestsThatTriggerErrors="true"
```

Comment:
This is a welcome improvement, but if you retarget to 4.2.x, this change should still go to 4.1.x or lower.